packages/core/src/scripts/build.ts

```diff
@@ -60,14 +60,12 @@ const build = async ({
   if (mode === "production") await remove(join(outDir, "bundling"));
 };
 
-(process as NodeJS.EventEmitter).on("unhandledRejection", (error: Error) => {
-  console.error(error);
-  process.exit(1);
-});
+// (process as NodeJS.EventEmitter).on("unhandledRejection", (error: Error) => {
```

Comment:
You can remove all the comments of the `dev`, `build` and `serve` files to keep them clean.